taqueria-analytics/index.ts

```diff
@@ -16,12 +16,15 @@ const toRequestURI = () =>
 	`https://www.google-analytics.com/mp/collect?measurement_id=${MEASUREMENT_ID}&api_secret=${API_SECRET}`;
 
 export const inject = ({ taqVersion, taqBuild, fields, getMachineId, operatingSystem, fetch, ...deps }: Deps) => {
-	const events: StoredEvent[] = [];
+	let events: StoredEvent[] = [];
```

Comment:
why do we need to mutate `events`?

Reply:
Yes, so that we can empty the array after sending events in the request to GA. The events array is meant to represent the list of events _to_ send.